spec/lib/protobuffness/compiler_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "protobuffness/compiler"
+
+RSpec.describe Protobuffness::Compiler do
+  let(:request_bytes) { File.read("spec/fixtures/sally_request.bin", :mode => "rb") }
```

Comment:
I mostly wrote this test because I wanted to be able to throw a `binding.pry` into the compiler code. During normal compilation this isn't possible since the `protoc` compiler invokes our compiler for us, no interactive shell is available.